$OpenBSD: patch-c2hs_c2hs-config_in,v 1.1 2004/05/09 07:46:12 dons Exp $

Have the old (non-package) configuration tool report the correct paths
to the library

--- c2hs/c2hs-config.in.orig	2002-07-12 14:39:46.000000000 +1000
+++ c2hs/c2hs-config.in	2004-05-09 15:20:56.000000000 +1000
@@ -74,7 +74,7 @@ fi
 if test x$has_pkg = xyes; then
   if test "$echo_cflags" = "yes" -o "$echo_libs" = "yes"; then
     if test "$pkg_added" != "yes"; then
-      echo -n "-package-conf $pkg_conf "
+      echo -n ""
     fi
     echo "-package c2hs"
   fi
